app/Models/Product.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    protected $primaryKey = 'id';
+    protected $table = 'tbl_product';
+    
+    protected $fillable = [
+    	'brand_id',
+        'product_name',
+        'product_price',
+        'product_image',
```

Comment:
```suggestion
    protected $fillable = [
        'brand_id',
        'product_name',
        'product_price',
        'product_image',
```
đoạn này vẫn đang dùng tab nè, chuyển sang dùng 4 dấu cách cho m nha